python/gcp_workspace_table_to_dataset_ingest.py

```diff
@@ -114,63 +105,69 @@ def get_args() -> argparse.Namespace:
     )
     tdr = TDR(request_util=request_util)
 
-    # Get sample metrics from Terra
-    sample_metrics = terra_workspace.get_gcp_workspace_metrics(entity_type=terra_table_name)
-    logging.info(f"Got {len(sample_metrics)} samples")
-
-    # Convert sample dict into list of usable dicts for ingestion
-    updated_metrics = ConvertTerraTableInfoForIngest(
-        table_metadata=sample_metrics,
-        tdr_row_id=primary_key_column_name,
-        columns_to_ignore=COLUMNS_TO_IGNORE
-    ).run()
-
-    # Use only specific sample ids if provided
-    if records_to_ingest:
-        updated_metrics = [
-            metric for metric in updated_metrics if metric[primary_key_column_name] in records_to_ingest
-        ]
-
-    if FILTER_EXISTING_IDS:
-        # Filter out sample ids that are already in the dataset
-        filtered_metrics = FilterOutSampleIdsAlreadyInDataset(
-            ingest_metrics=updated_metrics,
-            dataset_id=dataset_id,
-            tdr=tdr,
-            target_table_name=target_table_name,
-            filter_entity_id=primary_key_column_name,
-        ).run()
-    else:
-        filtered_metrics = updated_metrics
-
-    table_info_dict = {
-        target_table_name: {
-            "table_name": target_table_name,
-            "primary_key": primary_key_column_name,
-            "ingest_metadata": filtered_metrics,
-            "file_list": False,
-            "datePartitionOptions": None
-        }
-
-    }
-    SetUpTDRTables(tdr=tdr, dataset_id=dataset_id, table_info_dict=table_info_dict).run()
     GetPermissionsForWorkspaceIngest(
         terra_workspace=terra_workspace,
         dataset_info=tdr.get_dataset_info(dataset_id=dataset_id),
         added_to_auth_domain=True,
     ).run()
 
-    BatchIngest(
-        ingest_metadata=filtered_metrics,
-        tdr=tdr,
-        target_table_name=target_table_name,
-        dataset_id=dataset_id,
-        batch_size=BATCH_SIZE,
-        bulk_mode=bulk_mode,
-        cloud_type=CLOUD_TYPE,
-        update_strategy=update_strategy,
-        waiting_time_to_poll=WAITING_TIME_TO_POLL,
-        test_ingest=TEST_INGEST,
-        load_tag=f"{billing_project}_{workspace_name}-{dataset_id}",
-        file_list_bool=False
-    ).run()
+    for terra_table_name in terra_tables:
+        target_table_name = terra_table_name
+
+        # Get sample metrics from Terra
+        sample_metrics = terra_workspace.get_gcp_workspace_metrics(entity_type=terra_table_name)
+        entity_metrics = terra_workspace.get_workspace_entity_info()
```

Comment:
Should this be outside of for loop so you don't hit this multiple times?

Reply:
I think we want this for each entity type (i.e. terra_table_name) right?

Reply:
actually the `get_workspace_entity_info` can be called outside of the for loop, you're right! But I think `get_gcp_workspace_metrics` has to be called within, no?

Reply:
Yea, getting the individual metrics should happen within the for loop